docs/releasenotes/2.6.0.rst

```diff
@@ -0,0 +1,22 @@
+2.6.0
+-----
+
+Security
+========
+
+:cve:`2014-3589`: Fix DOS attack
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``PIL/IcnsImagePlugin.py`` in Python Imaging Library (PIL) and Pillow before 2.3.2 and
+2.5.x before 2.5.2 allows remote attackers to cause a denial of service via a crafted
+block size.
+
+Found and reported by Andrew Drake of dropbox.com
```

Comment:
Feel free to leave it in if you want, but why does 2.6.0 contain release notes for something that was fixed and documented in 2.5.2? Do you think we should be repeating security notes in minor releases from intermediate patch releases?

Reply:
https://github.com/python-pillow/Pillow/pull/7877#discussion_r1526108860

Reply:
In addition to the discussion with @hugovk linked to above (because It's almost impossible to find previous conversations!) I would say… NO. The release policy has changed since then, and we probably wouldn't do that now. BUT, because this is a historical reference, we're not so much _doing that now_ as we are _clarifying that we did it then._

Alternatively, if we were to make a new policy that all major changes contained release notes for security fixes from previous releases, then YES. However I'm not focused on that and I'd have to go through all the old release notes to see if there is any precedent and I'm not planning to do that.

Reply:
Agreed, I think it's okay to repeat in this instance: there were multiple releases with this.